inluxclient/client.go

```diff
@@ -71,3 +93,77 @@ func New(params Params) (*Client, error) {
 	}
 	return c, nil
 }
+
+// makeAPICall issues an HTTP request to InfluxDB server API url according to parameters.
+// Additionally, sets Authorization header and User-Agent.
```

Comment:
Shouldn't we also set `Content-Type` to `application/json` by default when body is non-nil

Reply:
Future `WritePoints` could use this function and it will set a different content-type

Reply:
OK, setting the content type is always the responsibility of the caller.